.github/workflows/release.yml

```diff
@@ -80,4 +89,4 @@ jobs:
       - name: Publish to NPM Registry 🌐
         run: |
           npm set "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}"
-          npm publish
+          npm publish --provenance --access public
```

Comment:
`--provenance` enables provenance.
`--access public` is recommended by NPM. Not sure why, it seems to just make it public NPM package, so went with it.